packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -107,3 +123,31 @@ export function UserHeaderItem(props: PropsWithChildren<UserHeaderItemProps>) {
         </AppHeaderDropdown>
     );
 }
+
+const StyledAvatar = styled(Avatar)<StyledAvatarProps>`
+    border: 1px solid ${({ theme }) => theme.palette.grey[400]};
```

Comment:
You can remove this, as it's set in the `&&` selector below

```suggestion
```

Reply:
baf5ba0a782f41920369ae8f617ce8a088d1f7e3